### PR TITLE
[TW-1447] VS Code extension updates

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
SEO and other improvements

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/getting-started/basics/about-vs-code-extension.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "About the Postman VS Code extension"
 updated: 2023-09-20
+search_keyword: "vs code extension, vs code plugin, vs code rest client, vs code api"
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/basics/about-vs-code-extension.md:4]:
```suggestion
search_keyword: "vs code extension, vs code plugin, vs code rest client, vs code api, VSC extension, visual studio extension"
```

File: src/pages/docs/getting-started/installation/installation-and-updates.md [hunk 2]
```diff
@@ -140,6 +141,10 @@ The Postman web app is under active development. There are features you can only
 * **Postman Interceptor** - [Interceptor](/docs/sending-requests/capturing-request-data/interceptor/) syncs cookies from your browser to the Postman desktop app and captures network requests directly from the browser. It's not possible to use Interceptor with the Postman web app.
 * **Lightweight Postman API Client** - You can use the [lightweight API Client](/docs/getting-started/basics/using-api-client/) to send requests from the Postman desktop app when you aren't signed in to Postman.
 
+## Using the Postman VS Code extension
+
+You can use Postman as a Visual Studio Code extension to test and collaborate on your APIs in the same application you're using to develop. With the [Postman VS Code extension](/docs/getting-started/basics/about-vs-code-extension/), you can send API requests, manage collections and environments, troubleshoot requests, and more directly from VS Code. To install the VS Code extension, visit the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Postman.postman-for-vscode).
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/installation/installation-and-updates.md:146]:
```suggestion
You can use the Postman Visual Studio Code extension to test and collaborate on your APIs in the same application you're using to develop. With the [Postman VS Code extension](/docs/getting-started/basics/about-vs-code-extension/), you can send API requests, manage collections and environments, troubleshoot requests, and more directly from VS Code. To install the VS Code extension, visit the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Postman.postman-for-vscode).
```

**Review by jkonrath-postman — APPROVED** [review]
Two little things, but looks good!

File: src/pages/docs/getting-started/installation/installation-and-updates.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/installation/installation-and-updates.md]: Commit based on your feedback: https://github.com/postmanlabs/postman-docs/pull/5162/commits/e5a0211837bb905c0dcee8799687908261ed450f

**Pull request merged**